# Message structure for wheel angular velocities for Dynamixel series AX
# FR - Front right / FL - Front left / BR - Back right / BL - Back left
# Each wheel can only have positive angular velocities

uint32 FR_vel
uint32 FL_vel
uint32 BR_vel
uint32 BL_vel

